src/pizzeria/pizzeria.ts

```diff
@@ -0,0 +1,57 @@
+import { ERROR_MESSAGES } from '../data/messages';
+import { Order } from '../order/order';
+
+export class Pizzeria {
+  private orders: Order[] = [];
+
+  constructor(
+    public name: string,
+    public address: string,
+    public workingHours: string,
+    public initialOrders?: Order[]
+  ) {
+    if (initialOrders) {
+      if (!Array.isArray(initialOrders)) {
+        throw new Error(ERROR_MESSAGES.INVALID_INITIAL_ORDERS);
+      }
+      this.orders = [...initialOrders];
+    }
```

Comment:
Если не ошибаюсь, так тоже возможно
```suggestion
    if (initialOrders && !Array.isArray(initialOrders)) {
        throw new Error(ERROR_MESSAGES.INVALID_INITIAL_ORDERS);
      }
      this.orders = [...initialOrders];
    }
```

Reply:
@VladimirLQA так во втором случае если initialOrders будет undefined у нас же сломается код на 16-й строке. Разве нет?

Reply:
Да согласен, провтьікал